CMakeLists.txt

```diff
@@ -113,6 +113,8 @@ else()
     set(SPDLOG_WCHAR_FILENAMES OFF CACHE BOOL "non supported option" FORCE)
 endif()
 
+set(SPDLOG_ASYNC_OWNING_SOURCELOC_STRINGS BOOL "Async loggers take owership of source_loc strings" OFF)
```

Comment:
You should use `option()` instead of `set()`.

Also, it seems halfway to define a new option in this position.
Why not define it where other options are defined together (e.g., after `option(SPDLOG_DISABLE_DEFAULT_LOGGER ...)`)?